numpy.testing.decorators module has been removed from the new version of numpy. Add this back to be compatible with scipy.


diff --git a/numpy/testing/decorators.py b/numpy/testing/decorators.py
new file mode 100644
index 000000000..690a9a2c6
--- /dev/null
+++ b/numpy/testing/decorators.py
@@ -0,0 +1 @@
+from numpy.testing._private.decorators import *
